driver.go

```diff
@@ -36,7 +35,7 @@ func (d ipfsDriver) Create(r volume.Request) volume.Response {
 		return volume.Response{}
 	}
 
-	volumePath := filepath.Join(d.mountPoint, volumeName)
+	volumePath := volumeName
```

Comment:
^^ True that...
Maybe it would be better to add a second plugin:
`docker run -it --rm -v QmdiHSUmmhMj9tNx5tKqsoJcen2jeUHjKwGeR8EniaZJ4u:/data --volume-driver=ipns busybox cat data`